### ISSUE TITLE
make titlebar component responsive

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#122]
(after #121)

**Comment by trvrb** [comment]
@jameshadfield ---

I tried to do this in #189, but ran into an issue with making putting `browserDimensions` into redux in `index.js` instead of `app.js` (one level up). I got the redux boilerplate working, but couldn't figure out how to get `dispatch` going.

I thought it would be easiest to make titlebar responsive by making `browserDimensions` available everywhere and then just querying `browserDimensions.width` to see whether to draw the "nextstrain" logotype or to draw the circular logo.

If this sounds reasonable to you, would you mind taking a look at getting `browserDimensions` working throughout static content?

**Comment by jameshadfield** [comment]
No problem - this is done in branch `browser-dimensions`

If it looks good then you can merge into your working branch I think

**Comment by trvrb** [comment]
Awesome. This is super clean. Thanks James.

**Comment by jameshadfield** [comment]
Browser dimensions fixed in 3af93988461fed791042a13c4045680351a3ad3f
